portfolio/src/main/webapp/dartmouth.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tal's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <ul>
+        <li><a class="active" href="index.html">Home</a></li>
+        <li><a href="hobbies.html">Hobbies</a></li>
+        <li><a href="dartmouth.html">School/Profession</a></li>
+        <li><a href="blog.html">Blog</a></li>
+
+    </ul>
+    <p> </p>
+
+   
+    <div id="content">
+      <h1>My Professional Life</h1>
+      <h2>Dartmouth College</h2>
+      <p>I am a rising sophomore at Dartmouth College. Instead of 
+          refering to ourselves as sophomores, we call ourselves by our class's
+          graduation year, '23s. At Dartmouth, I have yet to declare a major. I've 
+          taken many CS courses and a math course, both of which I've enjoyed,
+          but I also want to try out economics and earth science, specifically
+          oceanography. At school, I am involved in the Women in Computer Science, 
+          or WICS club as well as the Surf Club. I'm hopeful that I'll get to return 
+          to campus sometime soon.</p>
+        <h2>Google</h2>
```

Comment:
nit:  I would suggest watching out for indentation.  [2 spaces per block](https://google.github.io/styleguide/htmlcssguide.html#Indentation) I believe is recommended for HTML.  It'll get hard to keep track of which closing tags goes with which opening tags once the file grows huge.

Might be able to cheat here a bit though.  Most editors can automatically format this for you. eg. VS Code + Beautify